test/test_jit.py

```diff
@@ -14971,6 +15055,57 @@ def foo():
 
         self.assertEqual(foo(), torch.jit.script(foo)())
 
+    def test_optional_tuple(self):
+        def fn(x=None):
+            # type: (Optional[Tuple[int, int]]) -> Tuple[int, int]
+            if x is None:
+                new_x = (1, 2)
+            else:
+                new_x = x
+            return new_x
+
+        self.checkScript(fn, ((3, 4),))
+        self.checkScript(fn, ())
+
+    def test_namedtuple_redefine(self):
+        global _1, _2
+        _1 = namedtuple('GoogLeNetOutputs', ['logits', 'aux_logits2', 'aux_logits1'])
+        _2 = namedtuple('GoogLeNetOutputs', ['different'])
```

Comment:
Is this code copied from serialized model? The `_1` and `_2` seems a bit odd, please name them properly.

Reply:
Looks like this was a `rebase` issue. The original code is [here](https://github.com/pytorch/pytorch/blob/master/test/jit/test_typing.py#L535). I'll delete this